security/subsystem/src/main/java/org/jboss/as/security/SecurityDomainResourceDefinition.java

```diff
@@ -63,9 +64,12 @@
  */
 class SecurityDomainResourceDefinition extends SimpleResourceDefinition {
 
+    static final String INFINISPAN_CACHE_TYPE = "infinispan";
+    static final RuntimeCapability<Void> INFINISPAN = RuntimeCapability.Builder.of("org.wildfly.security.domain.infinispan", true).build();
```

Comment:
@pferraro OK, good. Would #9742 work for you?

There's 1 test failure with that PR, so if resolving that or otherwise getting approval for that to be merged will hold you up, I'm happy to merge this and carry on later. I've just long known we need a non-private cap for these security domain resources, so this PR kind of sparked me to move on it.